easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,47 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+#
+# Building this in interactive session will result in freeze during either
```

Comment:
Not true, works without a problem in interactive session

Reply:
Unless this refers to an interactive Slurm job session, if so it should be made clearer.

Reply:
It does refer to the latter, I'll clarify the comment.